bskyweb/static/.well-known/apple-app-site-association

```diff
@@ -11,5 +8,8 @@
         ]
       }
     ]
-  }
+  },
+  "appclips": {
+    "apps": ["B3LX46C5HS.xyz.blueskyweb.app.AppClip"]
+  },
```

Comment:
Note this is invalid JSON (trailing comma), is that OK?

Reply:
```suggestion
  }
```